ios/Headless Universal Checkout/RNTPrimerHeadlessUniversalCheckout.swift

```diff
@@ -106,17 +106,13 @@ class RNTPrimerHeadlessUniversalCheckout: RCTEventEmitter {
                                              errorCallback: @escaping RCTResponseSenderBlock,
                                              successCallback: @escaping RCTResponseSenderBlock)
     {
+        guard let image = PrimerHeadlessUniversalCheckout.getAsset(for: paymentMethodTypeStr, assetType: PrimerAsset.ImageType(rawValue: assetTypeStr)!) else {
```

Comment:
are we 💯 sure that we'll have a `PrimerAsset` here? If not, would rather check whether I get the `assetType` first:
```suggestion
        guard let assetType = PrimerAsset.ImageType(rawValue: assetTypeStr), let image = PrimerHeadlessUniversalCheckout.getAsset(for: paymentMethodTypeStr, assetType: assetType) else {
```